project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.5
+sbt.version=0.13.13-RC3
```

Comment:
Not actually, I just wanted to avoid as many future breaking changes as possible. 0.13.12 is totally fine to me.